openstack/cbshelf/templates/cronjob.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+    namespace: {{ .Release.namespace }}
+    name: volume-backup
+spec:
+  schedule: {{.Values.volume.backup.schedule}}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          affinity:
+            podAffinity:
+                requiredDuringSchedulingIgnoredDuringExecution:
+                - labelSelector:
+                    matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                      - {{ tpl .Values.volume.backup.app $ }}
+                  topologyKey: "kubernetes.io/hostname"
+          volumes:
+          - name: backup
+            persistentVolumeClaim:
+                claimName: {{ .Values.volume.backup.pvcName }}
+          containers:
+            - name: backup
+              image: 'keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/restic/restic:0.15.1'
```

Please use templating here, in order to automatically pick up the change if we ever need to move this account.
```suggestion
              image: '{{ .Values.global.dockerHubMirror }}/restic/restic:0.15.1'
```